### DISCUSSION TITLE
Calculate the Power

### DISCUSSION THREAD

**Discussion by FatihOnder81** [sonsuzus/sonsuzus#403] [Soru Cevap]
A ve B olmak üzere iki sayı verilmiştir . A B'nin değerini hesaplayın . Çıktı çok büyük olabilir, bu nedenle modülü 10^9+7 yanıtını yazdırın .

Giriş: Giriş, boşlukla ayrılmış iki A ve B
tamsayısını içerir .

Çıktı: A B modülü 10^9+7'nin
değerini yazdırın .

Kısıtlamalar:
Test Dosyası 1 - 5:
1<= A <=10
1<= B <=10

Test Dosyası 6 - 10: 1
<= A <=10 6 1<= B <=10 10

**Comment by EfeMertIrez** [comment]
```
import math 
s,ss=map(int,input().split())
m=(10**9)+7
print(pow(s,ss,m))
```
